VowelOrConsonant.java

```diff
@@ -0,0 +1,30 @@
+/*Write a Java program that takes the user to provide a single character from the alphabet. Print Vowel or Consonant, depending on the user input. If the user input is not a letter (between a and z or A and Z), or is a string of length > 1, print an error message.*/
+ 
+import java.util.Scanner;
+public class VowelOrConsonant
+{
+	
+	public void checkVowelOrConsonant(char ch)
+	{
+		if(ch=='a'||ch=='A'||ch=='e'||ch=='E'||ch=='i'||ch=='I'||ch=='o'||ch=='O'||ch=='u'||ch=='U')

```

Comment:
Please follow coding guidelines and put parenthesis for if and else block. Provide spaces around operators